packages/x-charts/src/RadarChart/RadarAxis/RadarAxis.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import composeClasses from '@mui/utils/composeClasses';
+import { useTheme } from '@mui/material/styles';
+import { useRadarAxis, UseRadarAxisParams } from './useRadarAxis';
+import { getLabelAttributes } from './RadarAxis.utils';
+import { getRadarAxisUtilityClass, RadarAxisClasses } from './radarAxisClasses';
+
+export interface RadarAxisProps extends UseRadarAxisParams {
+  /**
+   * Defines how label align with the axis.
+   * - 'horizontal': labels stay horizontal and their placement change with the axis angle.
+   * - 'rotated': labels are rotated 90deg relatively to their axis.
+   * @default 'horizontal'
+   */
+  labelOrientation?: 'horizontal' | 'rotated';
+  /**
+   * The labels text anchor or a function returning the text anchor for a given axis angle (in degree).
+   */
+  textAnchor?: string | ((angle: number) => string);
+  /**
+   * The labels dominant baseline or a function returning the dominant baseline for a given axis angle (in degree).
+   */
+  dominantBaseline?: string | ((angle: number) => string);
```

Comment:
Cant we provide the proper values that should be used here? `string` may be too generic

Reply:
Not yet but I'm working on it :)
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/73510/files